src/core/client/admin/components/StoryInfoDrawer/RescrapeStory.css

```diff
@@ -0,0 +1,3 @@
+.button {
+  width: 109px;
```

Comment:
This looks suspiciously like it might be trying to fit specifically to a width that could probably be laid out better using a `<Flex>` wrapper. Would that be possible?